src/components/Dashboards/CurrentDashboards.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 
-export default function CurrentDashboards() {
+// comment
+export function CurrentDashboards() {
```

Comment:
this is a personal preference, but I definitely tend to lead my juniors that I coach away from default exports. When working in larger codebases, it is sometimes unclear what you're importing (especially when there are multiple exports in a file).  You may end up with something like this:

```js
import CurrentDashboards, { AN_IMPORTANT_CONSTANT } from '../Dashboards/CurrentDashboards';
```

Another (albeit small) issue, is if you ever want to see everywhere `CurrentDashboards` is used, you'd have to search for the filename - this is because with default exports people can do this:

```
import isntThisATerribleImportName from '../Dashboards/CurrentDashboards';

/// bunch of code here

return ( <isntThisATerribleImportName /> );
```

and be fair game. With named exports, the engineers using the components are **forced** to import it by name:

```
import { CurrentDashboards } from '../Dashboards/CurrentDashboards';
```

if they need to rename for some reason, they can still do that, **without ruining your ability to search it later:**

```
import { CurrentDashboards as MyStupidImportName } from '../Dashboards/CurrentDashboards';

/// code

return ( <MyStupidImportName /> );
```